sushy_oem_idrac/resources/manager/manager.py

```diff
@@ -83,6 +108,98 @@ class DellManagerExtension(oem_base.OEMResourceBase):
     def import_system_configuration_uri(self):
         return self._actions.import_system_configuration.target_uri
 
+    @property
+    def job_service_uri(self):
+        return self._links.job_service.resource_uri
+
+    @property
+    def get_jobs_uri(self):
+        return self._links.jobs.resource_uri
+
+    @property
+    def get_remote_service_api_uri(self):
+        return ('%s%s' %
+                (self._links.lc_service.resource_uri,
+                self.REMOTE_API_STATUS_URI))
+    @property
+    def get_idrac_reset_uri(self):
+        return ('%s%s' %
+                (self._links.idrac_card_service.resource_uri,
+                self.IDRAC_RESET_URI))
+
+    def delete_jobs(self, task, job_ids=['JID_CLEARALL']):
+
+        if job_ids is None:
+            return None
+
+        delete_job_queue_uri = '%s%s' % (self.job_service_uri, self.CLEAR_JOBS_URI)
+        for job_id in job_ids:
+            payload = {'JobID':job_id}
+
+            try:
+                delete_job_response = asynchronous.http_call(
+                        self._conn, 'post',
+                        delete_job_queue_uri,
+                        headers = self.HEADERS,
+                        data = payload,
+                        verify = False)
+                return delete_job_response
+
+            except (sushy.exceptions.ServerSideError,
+                    sushy.exceptions.BadRequestError) as exc:
+                LOG.error('Dell OEM clear job queue failed, on node %(node)s'
+                           'Reason : %(reason)s' %
+                           {'node': task.node.uuid, 'reason': exc})
+                raise
+
+    def reset_idrac(self, task):
+        payload = {"Force":"Graceful"}
```

Comment:
Can you add a "force" parameter to this method that defaults to "Graceful" and in the payload:
{"Force": force}

Reply:
Done. Added force parameter that defaults to "Garceful".